chrome-launcher/chrome-launcher.ts

```diff
@@ -45,24 +48,79 @@ export interface ModuleOverrides {
   spawn?: typeof childProcess.spawn;
 }
 
-export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
-  opts.handleSIGINT = defaults(opts.handleSIGINT, true);
+/**
+ * LaunchManager handles launch retries and SIGINT binding
+ */
+export class LaunchManager {
+  private instance?: Launcher;
+  private opts: Options;
+  private isSigintBound: boolean;
+  private maxLaunchAttempts: number;
+  private attempts: number;
+
+  constructor() {
+    this.isSigintBound = false;
+    this.maxLaunchAttempts = 3;
+    this.attempts = 1;
```

Comment:
I'm a little confused the fact it's already `1`, how about initializing to `0` and treating it as 'failedAttempts'?